File: THE CPP/prime test/factors.cpp
#include <iostream>
#include <cmath>
using namespace std;

int main()
{
    int number;
    bool isfactor = 0;
    cout << "enter number: ";
    cin >> number;
    for (int i = 2; i <= sqrt(number); i++)
    {
        if (number % i == 0)
        {
            isfactor = 1;
        }
        else
        {
            isfactor = 0;
        }  
        if (isfactor == 1)
        {
            cout<<i<<"  ";
        }
        
    }

    return 0;
}

File: PYTHON/factors.py
import math 
num = int(input('Enter number: '))

# find factor of number
print('The factors of', num, 'are:')
rootnum =(int)( math.sqrt(num) + 1)
for i in range(1, (rootnum), 2):
    if(num % i) == 0:
        print(i, end=' ')
